modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -30,7 +30,7 @@ You must have the exact list of container image references used by the {product-
 +
 [source,terminal]
 ----
-$ VERSION=$(rpm -q --queryformat='%{VERSION}-%{RELEASE}.%{ARCH}' microshift)
+$ VERSION=$(dnf list | awk "/^microshift\./ {print \$2;}")
```

Comment:
This can return multiple hits, right? How do we make sure we're getting the latest one?

Reply:
`dnf list` will only return the latest version per package, unless one uses the `--showduplicates` option.

Reply:
Shouldn't we use `sudo` because without it I get permission error during repo metadata download
```
$ dnf list | awk "/^microshift\./ {print \$2;}"
Errors during downloading metadata for repository 'rhui-codeready-builder-for-rhel-8-x86_64-rhui-debug-rpms':
  - Curl error (58): Problem with the local SSL certificate for https://rhui.googlecloud.com/pulp/mirror/content/dist/rhel8/rhui/8/x86_64/codeready-builder/debug [could not load PEM client certificate, OpenSSL error error:0200100D:system library:fopen:Permission denied, (no key found, wrong pass phrase, or wrong file format?)]
Error: Failed to download metadata for repo 'rhui-codeready-builder-for-rhel-8-x86_64-rhui-debug-rpms': Cannot prepare internal mirrorlist: Curl error (58): Problem with the local SSL certificate for https://rhui.googlecloud.com/pulp/mirror/content/dist/rhel8/rhui/8/x86_64/codeready-builder/debug [could not load PEM client certificate, OpenSSL error error:0200100D:system library:fopen:Permission denied, (no key found, wrong pass phrase, or wrong file format?)]
```

with `sudo` it works
```
$ sudo dnf list | awk "/^microshift\./ {print \$2;}"
4.12.1-202301261525.p0.g3db9e81.assembly.4.12.1.el8
```

Reply:
Strange, didn't get this before. Either way: fixed.